ci-operator/step-registry/image-based-install-operator/gather/image-based-install-operator-gather-commands.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ image based install operator gather command ************"
+
+source "${SHARED_DIR}/packet-conf.sh"
+
+ssh "${SSHOPTS[@]}" "root@${IP}" bash - << EOF
+
+# prepending each printed line with a timestamp
+exec > >(awk '{ print strftime("[%Y-%m-%d %H:%M:%S]"), \$0 }') 2>&1
+
+set -xeo pipefail
+
+cd /root/dev-scripts
+source common.sh
+source utils.sh
+source network.sh
+
+oc get baremetalhost ostest-extraworker-0 -n openshift-machine-api -o yaml
+oc get clusterdeployment ibi-cluster -n ibi-cluster -o yaml
+oc get imageclusterinstall ibi-cluster -n ibi-cluster -o yaml
+oc logs --tail=-1 -l app=image-based-install-operator -n image-based-install-operator -c manager
+oc logs --tail=-1 -l app=image-based-install-operator -n image-based-install-operator -c server
```

Comment:
Better if you save it to $ARTIFACT_DIR e.g.
oc get imageclusterinstall ibi-cluster -n ibi-cluster -o yaml >  $ARTIFACT_DIR/imageclusterinstall.yaml